### DISCUSSION TITLE
Bayesian regression entry methods and the difference between the predictor matrix and formulas

### DISCUSSION THREAD

**Discussion by Max-Lovell** [amices/mice#440] [Q&A]
Hi,

I have two questions regarding the entry methods of the bayesian regression (norm) option. 
How is this handled when using a predictor matrix - is it forced entry down the list of variables or simultaneous? If so, could regression entry order by be specified using formulas? I could then calculate stepwise entry orders earlier.

A final P.S. if anyone if anyone has any opinions or relevant papers to hand, I'm stuck between handling the items of my scales as single scale blocks or doing each item individually? I have missing data at post-test and am predicting this from pre-test. At this point I am handling it item by item - predictors in my regression equations will be the same, although outcomes will differ.

Any help much appreciated! thanks.

**Comment by stefvanbuuren** [comment]
There is currently no stepwise modelling in `mice.impute.norm()`, but there is one in preparation using lasso <https://github.com/amices/mice/pull/438>.

**Reply by Max-Lovell** [reply]
Thanks for your reply Stef! Could I ask how entry is handled in the impute.norm method, and if this changes when specifying blocks and formulas vs. a predictor matrix?

**Reply by thomvolker** [reply]
The default order of imputation specified in `mice` is from left to right. However, you can easily change the sequence by specifying the `visitSequence` parameter within the `mice::mice` function. You can do this once, specifying the same sequence for every variable that is to be imputed, or for every variable separately.

Hope this helps.

**Reply by stefvanbuuren** [reply]
Sorry, I do not quite understand your question(s). Could you perhaps illustrate the problem with a small example?
